### ISSUE TITLE
run_eval got this issue

### ISSUE THREAD

**Issue by dosinsane** [huggingface/distil-whisper#165]
UserWarning: `do_sample` is set to `False`. However, `top_k` is set to `0` -- this flag is only used in sample-based generation modes. You should set `do_sample=True` or unset `top_k`.
The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

Can somebody help me with this issue 
this happen when i run author code, the hi language i found WER is so high that number is 100 up , so i try to finds why cause this issue happen and my terminal response me this, i try my best but i still can't where the bugs is and how to address this so plz some body help me Thanks
